types/types.go

```diff
@@ -414,11 +415,11 @@ func GetShareManagerImageChecksumName(image string) string {
 }
 
 func GetShareManagerPodNameFromShareManagerName(smName string) string {
-	return LonghornLabelShareManager + "-" + smName
+	return shareManagerPrefix + "-" + smName
 }
 
 func GetShareManagerNameFromShareManagerPodName(podName string) string {
-	return strings.TrimPrefix(podName, LonghornLabelShareManager+"-")
+	return strings.TrimPrefix(podName, shareManagerPrefix+"-")
```

Comment:
Since the prefix already includes, the - sign, the appending of the - is not longer needed.